tests/integration/test_lambda.py

```diff
@@ -411,6 +411,48 @@ def test_deletion_event_source_mapping_with_dynamodb(self):
         # clean up
         lambda_client.delete_function(FunctionName=function_name)
 
+    def test_put_subscribe_policy(self):
+        iam_client = aws_stack.connect_to_service('iam')
+        lambda_client = aws_stack.connect_to_service('lambda')
+        log_client = aws_stack.connect_to_service('logs')
+
+        role_name = 'role-{}'.format(short_uid())
+        assume_policy_document = {'Version': '2012-10-17',
+                                  'Statement': [
+                                      {
+                                          'Action': 'sts:AssumeRole',
+                                          'Principal': {
+                                              'Service': 'lambda.amazonaws.com'
+                                          },
+                                          'Effect': 'Allow',
+                                          'Sid': '',
+                                      }
+                                  ]
+                                  }
+
+        iam_client.create_role(
+            RoleName=role_name,
+            AssumeRolePolicyDocument=json.dumps(assume_policy_document)
+        )
+
+        testutil.create_lambda_function(
```

Comment:
To keep the test a bit more light-weight, we could simply create a log group manually here, instead of creating and invoking a Lambda function. In fact, since this PR is about CloudWatch Logs functionality (not strictly related to Lambda), I'd suggest that we move it to `test_logs.py`.